docs/getting-started/throwing-errors.md

```diff
@@ -0,0 +1,289 @@
+
+# Getting started: throwing errors
+
+Throwing good errors is key to producing a reliabile application. Most of Reliability Kit's packages rely on sensible errors being thrown if a fault is encountered in your app, and without reviewing your existing errors you won't get as much value from using it.
+
+  * [Types of error](#types-of-error)
+    * [Operational errors](#operational-errors)
+    * [Programmer errors](#programmer-errors)
+  * [Types of error in code](#types-of-error-in-code)
+    * [Operational errors in library code](#operational-errors-in-library-code)
+  * [What does a good error look like?](#what-does-a-good-error-look-like)
+    * [Making errors human readable](#making-errors-human-readable)
+    * [Making errors machine readable](#making-errors-machine-readable)
+    * [Adding more data](#adding-more-data)
+    * [Being specific](#being-specific)
+
+
+## Types of error
+
+Errors can be sorted into two broad categories, it's important to understand the difference because they should be handled differently by your application. These take inspiration from the [Joyent error handling documentation](https://www.joyent.com/node-js/production/design/errors), which is also worth a read.
```

Comment:
**nitpick**: comma splice (also added another comma)

```suggestion
Errors can be sorted into two broad categories. It's important to understand the difference, because they should be handled differently by your application. These take inspiration from the [Joyent error handling documentation](https://www.joyent.com/node-js/production/design/errors), which is also worth a read.
```